### PR TITLE
fix (DB): fix Ravager Spicimens overpopulation

### PR DESCRIPTION
**Pull request by TheDdraig**
## Changes Proposed:
-  This fixes the overpopulation of spawns for Ravager Spicimens

## Issues Addressed:
https://github.com/azerothcore/azerothcore-wotlk/issues/7327
- Closes 

## SOURCE:
Sniff Team

## Tests Performed:

- Build
- Performed

## How to Test the Changes:

1. go c 17199
2. You will see the area is overpopulated, for example 91 on azerothcore there is only 45 in this area on retail and upon checking these npcs are only in this particular area, ofc check and confirm if needed.

### CODE REVIEW

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
remove the notes after the semi colons. that is what is failing the code stye test
image

File: data/sql/updates/pending_db_world/rev_1643215013369974869.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
+-- Fixes Issue: https://github.com/azerothcore/azerothcore-wotlk/issues/7327
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643215013369974869');
+
+DELETE FROM `creature` WHERE `id1`='17199';
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1643215013369974869.sql:4]:
```suggestion
DELETE FROM `creature` WHERE `id1`=17199;
```

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1643215013369974869.sql:4]: Also add GUID to the delete statement

File: data/sql/updates/pending_db_world/rev_1643215013369974869.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+(61265, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3418.1714, -12752.151, 10.181131, 4.816044807434082031, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3418.1714 -12752.151 10.181131
+(61266, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3352.9028, -12686.875, 20.251678, 6.248052120208740234, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3352.9028 -12686.875 20.251678
+(61267, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3348.0188, -12749.719, 14.26492, 0.948191642761230468, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3348.0188 -12749.719 14.26492
+(61268, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3417.4426, -12716.594, 16.267937, 2.189768791198730468, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3417.4426 -12716.594 16.267937
+(61269, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3412.6267, -12687.893, 22.139761, 5.748758792877197265, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0), -- .go xyz -3412.6267 -12687.893 22.139761
+(61270, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3381.6501, -12779.417, 14.582455, 5.741998672485351562, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0); -- .go xyz -3381.6501 -12779.417 14.582455
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1643215013369974869.sql]: only the last note is causing the codestyle as it doesn't check for comments... If you want it you can do a `/* */` comment inside the );

Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1643215013369974869.sql]: oh we had this happen with a prior pr, its any not after the , or ; that is followed by a -- is what throws the error regardless if it is the last line or not in the code style test.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1643215013369974869.sql]: yea that is why you can write it
```sql
(61270, 17199, 0, 0, 530, 0, 0, 1, 1, 0, -3381.6501, -12779.417, 14.582455, 5.741998672485351562, 300, 10, 0, 1, 0, 1, 0, 0, 0, '', 0 /* .go xyz -3381.6501 -12779.417 14.582455-- .go xyz -3381.6501 -12779.417 14.582455*/);
```
For just the last line, as the CI always checks for `;` to be the last character on the last line of an SQL file

Comment (acidmanifesto) [reply data/sql/updates/pending_db_world/rev_1643215013369974869.sql]: that is good bit of info to know actually. hell i wish we knew about that bit about a month ago.

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1643215013369974869.sql (hunk 2 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1643215013369974869.sql]: I'd love for the CI to be able to ignore comments, but nobody knows how to do it yet

**Pull request merged**